lighthouse-core/test/runner-test.js

```diff
@@ -155,6 +156,63 @@ describe('Runner', () => {
         expect(runAuditSpy).toHaveBeenCalled();
       });
     });
+
+    it('serializes IcuMessages in gatherMode and is able to use them in auditMode', async () => {
```

Comment:
kind of a monster sized test because it does a gatherMode run _and_ an auditMode run, but I also didn't really want to split because it's annoying when tests have interdependencies, so 🤷

Reply:
It might make sense to split the three -G, -A, and `loadArtifacts` components separately since we're asserting the exact shape of the artifacts already? 

I would prioritize the e2e flow though like you did here anyway 🤷